report_sources/elr_review.Rmd

```diff
@@ -147,17 +147,18 @@ rate (*r*), current incidence (as % of previous peak) and signs of acceleration
 
 * if incid < 40%:
     + if r > high_r -> **M**
-	+ else -> NL
+    + else if r > 0.02 -> **L**
+	+ else -> Minimal
 * if 40% <= incid <= 80%:
```

Comment:
What style of coding is this? Just curious

Reply:
That was my attempt at pseudo-code, but I never learnt properly how to do this...